tsweb/tsweb.go

```diff
@@ -263,6 +265,18 @@ type HandlerOptions struct {
 	OnCompletion OnCompletionFunc
 }
 
+func setDefaultOpts(opts *HandlerOptions) {
```

Comment:
or method? "setDefaults"? (can drop opts, so callers are `opts.setDefaults()` and take advantage of method calls automatically taking the address when needed?)

Reply:
I've split the LogOptions and ErrorOptions out when constructing a logHandler/errorHandler, and used a `withDefaults()` method on both to return the options with defaults set.